jaxrs-api/src/main/java/jakarta/ws/rs/BadGatewayException.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2012, 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is wrong. Neither are you working for Oracle, nor was your code written in 2012. Please provide the truth here.

Reply:
Sure, I just used the same i found in BadRequestException. I should only remove this oracle part right?

Reply:
You could use something like:

    Copyright (c) 2019 Jeyvison Nascimento

Or a more generic one like:

    Copyright (c) 2019 Contributors to the Eclipse Foundation

Reply:
@jeyvison Here is the long answer: https://www.eclipse.org/projects/handbook/#ip-copyright-headers

Reply:
fixed.